dev-ruby/excon/excon-0.78.1.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+USE_RUBY="ruby25 ruby26 ruby27"
+
+RUBY_FAKEGEM_EXTRADOC="changelog.txt README.md"
+RUBY_FAKEGEM_GEMSPEC="${PN}.gemspec"
```

Comment:
We need to use the GEMSPEC here since we install from a tarball. We prefer not to use ${PN} here. This gemspec also triggers the common git warning that needs to be fixed: 

 * Running install phase for ruby25 ...
fatal: not a git repository (or any parent up to mount point /var/tmp)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).

Reply:
Ok upgraded